pkgs/applications/video/mpv/scripts/simple-mpv-webui.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub }:
+stdenv.mkDerivation {
+  pname = "simple-mpv-ui";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "open-dynaMIX";
+    repo = "simple-mpv-webui";
+    rev = "806d2a1465985ebe54e549ea94fec2f0d0c83dcf";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
Applied and amended. Thanks for the suggestion!